### PR TITLE
Inflight Magazine no. 08

### PR DESCRIPTION
**Pull request by eladb**

### CODE REVIEW

File: blog/2024-03-13-magazine-008.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+
+The language combines cloud infrastructure ***and*** runtime into a single programming model and **compiles to Terraform/CloudFormation and JavaScript** that are ready to deploy to AWS, GCP and Azure (see [support matrix](https://www.winglang.io/docs/standard-library/compatibility-matrix)).
+
+It is designed to be a **familiar and friendly** language to learn for developers who come from modern object-oriented background, so it will take you [5 minutes](https://github.com/winglang/workshop-react/blob/main/cheatsheet.md) to learn.
+
+Check out our [getting started tutorial](https://www.winglang.io/docs/) or hit the [Wing Playground](https://www.winglang.io/play/) for an online experience.
+
+### In today's issue
+
+- 🚀 [Deploying AWS resources and APIs inside a private VPCs](#)
```
Comment (eladb) [inline blog/2024-03-13-magazine-008.md:33]: Links

File: blog/2024-03-13-magazine-008.md [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+> NOTE: I spent like two hours trying to figure out the right emojis here so please take a minute to appreciate them :-)
+
+### Deploying AWS resources and APIs inside a private VPCs
+<a id="w1"></a>
+
+![](./2024-03-13-private-api.png)
+
+Users tell us they are looking for ways to deploy API-based services inside their VPCs. We've spent some time learning about [the challenges people have](https://www.winglang.io/blog/2024/02/15/private-api-gateways) and figured we can offer a awesome experience in Wing for this use case.
+
+We published this [blog post](https://www.winglang.io/blog/2024/02/09/private-api-gateway-aws) which describes how to configure the `tf-aws` platform to deploy your application's AWS resources and APIs inside a VPC, with support for creating a whole new VPC or deploying into a VPC that's already setup in your environment.
```
Comment (revitalbarletz) [inline blog/2024-03-13-magazine-008.md:54]: Do you mind changing the blog post to something like guide/tutorial?

File: blog/2024-03-13-magazine-008.md [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+
+> ”Wing made all the difference! After struggling for days with the CDK, I turned to Wing and had a fully functional Private API Gateway up and running within hours. Its simplicity and power are unparalleled. Thank you, Wing, for saving me time and headaches!”
+
+### A fresh look for the Wing Console map view
+<a id="w2"></a>
+
+
+We gave the map view in Wing Console some love, improved colors, cleaned up the UI and polished the interactions:
+
+![](./2024-03-13-console-new-look.mp4)
```
Comment (revitalbarletz) [inline blog/2024-03-13-magazine-008.md:68]: There is no image there

File: blog/2024-03-13-magazine-008.md [hunk 4]
```diff
@@ -0,0 +1,327 @@
...
+- 🔥 [Mutually referential type declarations](#w4)
+- 🛟 [The bang `!` operator](#w5)
+- 🏋️ [Ability to lift collection of inflight closures](#w6)
+- 🗄️ [`fs.appendFile()`](#w7)
+- ⚙ [Platform Parameters and AWS VPC configuration](#w8)
+- 🛌 [Ergonomic improvements for `cloud.Function` and `cloud.Api`](#w9)
+- 🎥 [Goodies from the Wingly Update](#w10)
+- 👯‍♀️ [Community Events](#w11)
+
+> NOTE: I spent like two hours trying to figure out the right emojis here so please take a minute to appreciate them :-)
```
Comment (revitalbarletz) [inline blog/2024-03-13-magazine-008.md:45]: :)

File: blog/2024-03-13-magazine-008.md [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+
+### Deploying AWS resources and APIs inside a private VPCs
+<a id="w1"></a>
+
+![](./2024-03-13-private-api.png)
+
+Users tell us they are looking for ways to deploy API-based services inside their VPCs. We've spent some time learning about [the challenges people have](https://www.winglang.io/blog/2024/02/15/private-api-gateways) and figured we can offer a awesome experience in Wing for this use case.
+
+We published this [blog post](https://www.winglang.io/blog/2024/02/09/private-api-gateway-aws) which describes how to configure the `tf-aws` platform to deploy your application's AWS resources and APIs inside a VPC, with support for creating a whole new VPC or deploying into a VPC that's already setup in your environment.
```
Comment (eladb) [inline blog/2024-03-13-magazine-008.md:56]:
```suggestion
We published a [guide](https://www.winglang.io/blog/2024/02/09/private-api-gateway-aws) which describes how to configure the `tf-aws` platform to deploy your application's AWS resources and APIs inside a VPC, with support for creating a whole new VPC or deploying into a VPC that's already setup in your environment.
```

File: blog/2024-03-13-magazine-008.md (hunk 2 above)
Comment (eladb) [reply blog/2024-03-13-magazine-008.md]: done

File: blog/2024-03-13-magazine-008.md (hunk 1 above)
Comment (eladb) [reply blog/2024-03-13-magazine-008.md]: Fixed

File: blog/2024-03-13-magazine-008.md (hunk 3 above)
Comment (eladb) [reply blog/2024-03-13-magazine-008.md]: Fixed. It's a video

File: blog/2024-03-13-magazine-008.md [hunk 6]
```diff
@@ -0,0 +1,319 @@
...
+
+[Ruslan Kazakov](https://github.com/rkazakov) from Sydney, Australia wrote about his experience in [a blog post](https://medium.com/bear-plus/secure-your-vpc-endpoint-for-api-gateway-with-winglang-235a5f8af072) :
+
+> ”Wing made all the difference! After struggling for days with the CDK, I turned to Wing and had a fully functional Private API Gateway up and running within hours. Its simplicity and power are unparalleled. Thank you, Wing, for saving me time and headaches!”
+
+### A fresh look for the Wing Console map view
+
+We gave the map view in Wing Console some love, improved colors, cleaned up the UI and polished the interactions:
+
+<ReactPlayer width="50%" height="50%" playing loop muted url={console_new_look}/>
```
Comment (revitalbarletz) [inline blog/2024-03-13-magazine-008.md:67]: Nothing is displayed...

Comment (eladb) [reply blog/2024-03-13-magazine-008.md:67]: See [preview](https://wing-docs-git-eladb-inflight-magazine-7-monada.vercel.app/blog/2024/03/13/magazine-008)

**Review by revitalbarletz — APPROVED** [review]

**Pull request merged**